allauth/socialaccount/providers/kakao/provider.py

```diff
@@ -23,5 +24,24 @@ class KakaoProvider(OAuth2Provider):
     def extract_uid(self, data):
         return str(data['id'])
 
+    def extract_common_fields(self, data):
+        email = None
+        if data.get("kaccount_email_verified"):
+            email = data.get("kaccount_email")
+
+        return dict(email=email)
+
+    def extract_email_addresses(self, data):
+        ret = []
+        email = data.get("kaccount_email")
+        verified = data.get("kaccount_email_verified")
+        if verified:
```

Comment:
Can you change this so that the email address is also added when verified is not true?